static/index.html

```diff
@@ -0,0 +1,262 @@
+              </div>
+              <button type="submit" id="add-order-btn">Submit</button>
+            </form>
+          </div>
+        </div>
+
+        <!-- Market dropdown -->
+
+        <div class="card">
+          <select name="market" id="market-selector">
+            <option value="">Select a market</option>
+          </select>
+          <div class="card-header">
+            <h2>Book</h2>
+          </div>
+          <div class="card-body">
+            <table class="data-table">
+              <thead>
+                <tr>
+                  <th>Price</th>
+                  <th>Amount</th>
+                  <th>User</th>
+                  <th>Timestamp</th>
+                </tr>
+              </thead>
+              <tbody>
+                <tr>
+                  <td>3000</td>
+                  <td>1</td>
+                  <td>Me</td>
+                  <td>Now</td>
+                </tr>
+              </tbody>
+            </table>
+          </div>
+
+          <div class="card-header">
+            <h2>My Order History</h2>
+          </div>
+          <div class="card-body">
+            <table class="data-table">
+              <thead>
+                <tr>
+                  <th>State</th>
+                  <th>Price</th>
+                  <th>Amount</th>
+                  <th>Time</th>
+                </tr>
+              </thead>
+              <tbody>
+                <tr>
+                  <td>Open</td>
+                  <td>3000</td>
+                  <td>1</td>
+                  <td>1 day ago</td>
+                </tr>
+              </tbody>
+            </table>
+          </div>
+        </div>
+
+        <div class="card">
+          <div class="card-header">
+            <h2>Settled Orders</h2>
+          </div>
+          <div class="card-body">
+            <table class="data-table">
+              <thead>
+                <tr>
+                  <th>Maker</th>
+                  <th>Taker</th>
+                  <th>Price</th>
+                  <th>Amount</th>
+                  <th>Time</th>
+                </tr>
+              </thead>
+              <tbody>
+                <tr>
+                  <td>0x0</td>
+                  <td>0x1</td>
+                  <td>3000</td>
+                  <td>1</td>
+                  <td>1 day ago</td>
+                </tr>
+              </tbody>
+            </table>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <script type="module">
+      import { createWalletClient, http, stringToHex, encodePacked, keccak256, toBytes, encodeAbiParameters } from "https://esm.sh/viem";
+      import { privateKeyToAccount } from "https://esm.sh/viem/accounts";
+      import { mainnet } from "https://esm.sh/viem/chains";
+
+      const client = createWalletClient({
+        chain: mainnet,
+        transport: http(),
+      });
+
+      const privateKey = "0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80";
```

Comment:
I was very surprised when I saw here a private key from some wallet with transactions. Maybe we can make the demo work with ethereum provider?

Reply:
It's the key that viem use: https://viem.sh/docs/accounts/local/privateKeyToAccount#generating-private-keys